### ISSUE TITLE
First steps

### ISSUE THREAD

**Issue by jw3126** [JuliaObjects/Accessors.jl#3]
Here is how I think we should proceed:

* Make the github repo public. @tkf can you review the package? Especially check if I inserted your name + mail correctly in all appropriate places.
* Register the package and put a note in Setfield.jl README that this is an experimental successor.
* Wait for Julia 1.6 to come out. Then announce the package on discourse and make it the official successor of Setfield.jl

**Comment by jw3126** [comment]
@tkf I would like to tag a first release (without discourse announcement yet). What do you think?

**Comment by jw3126** [comment]
@JuliaRegistrator register()

**Comment by jw3126** [comment]
@JuliaRegistrator register()

**Comment by jw3126** [comment]
@JuliaRegistrator register()

**Comment by tkf** [comment]
Hi, sorry for the long absence, and thanks so much for pushing this project forward!

If you prefer announcing it on discourse, I think that's good too. But a few things on my wish list off the top of my head are:

* Update the "downstream" packages like https://github.com/JuliaFolds/DataTools.jl and https://github.com/queryverse/VegaLite.jl

* Add the forward-diff tutorial (I know I've been saying this for a long time! :sob:)

* (Lower priority) Switch to the Tutorials/How-To/Explanation/Reference documentation system https://documentation.divio.com/. If you like the idea, I can try doing this for Accessors.jl around when I try the forward-diff tutorial.

**Comment by jw3126** [comment]
You are right, the docs could use some love. And it makes a lot of sense to improve them before announcing. If you want to tackle it that's great! Feel free to replace or delete any docs I wrote. 
I can update `DataTools.jl` it's a package I wanted to take a look at anyway.
